tests/TestCase/Command/WorkerCommandTest.php

```diff
@@ -191,17 +187,13 @@ public function testQueueProcessesJobWithProcessor()
             'levels' => ['notice', 'info', 'debug'],
         ]);
 
-        $callable = [WelcomeMailer::class, 'welcome'];
```

Comment:
I think this is the call format we need to retain as that is how all userland code is configured currently, and a hard break like that feels bad when we can easily avoid it.

Reply:
I added a data provider to test both ways of calling `QueueManager::push()`.